### DISCUSSION TITLE
Change SQLite-web Host IP Address

### DISCUSSION THREAD

**Discussion by ojo-ht** [hassio-addons/addon-sqlite-web#216] [Q&A] [answered]
I'm trying to extract data from hass OS running on RPi 3b+ and store specific samples onto my laptop for further processing. I am using the default SQlite database and was thinking I could simply install the Sqlite-web add-on and query the database from my laptop... 

However, I need to configure the sqlite-web add-on to host somewhere besides 127.0.0.1; I want this to be visible on my LAN... What can I add to the sqlite-web configuration (or maybe configuration.yaml) to alter this? 

If you understand what I'm trying to do and see a better path forward, please, your thoughts are appreciated!

**Comment by frenck** [comment, marked as answer]
> However, I need to configure the sqlite-web add-on to host somewhere besides 127.0.0.1; I want this to be visible on my LAN... What can I add to the sqlite-web configuration (or maybe configuration.yaml) to alter this?

It is available via the Home Assistant web interface only (Ingress feature).
